src/coreComponents/physicsSolvers/multiphysics/MultiphasePoromechanics.cpp

```diff
@@ -495,9 +499,6 @@ void MultiphasePoromechanics< FLOW_SOLVER >::mapSolutionBetweenSolvers( DomainPa
   /// After the flow solver
   if( solverType == static_cast< integer >( Base::SolverType::Flow ) )
   {
-    // save pressure and temperature at the end of this iteration
-    flowSolver()->saveIterationState( domain );
```

Comment:
@castelletto1 @joshua-white need help here to clarify the logic
what `saveIterationState` esentially does here is `pres_k=pres` which effectively modifies the computed porosity since fixed stress term disappears
i moved that assignment to the very end of sequential iteration but not sure about that